examples/assets/scripts/parsers/hdr-texture.js

```diff
@@ -46,16 +46,17 @@ Object.assign(Reader.prototype, {
  * @implements {pc.TextureParser}
  * @classdesc Texture parser for hdr files.
  */
-function HdrParser(registry, retryRequests) {
-    this.retryRequests = retryRequests;
+function HdrParser(registry, maxRetries) {
+    this.maxRetries = Math.max(maxRetries, 0) || 0;
```

Comment:
Shouldn't you set maxRetries as optional? And default it to 0?

Reply:
It is optional and defaults to 0

Reply:
Yeah, actually, true. But man, that code is confusing. Because `Math.max(undefined, 0)` is `NaN` and `Nan || 0` is `0`.

Reply:
Checking for undefined, NaN, negative values or other invalid values can become pretty verbose otherwise.. I preferred the shorter code which does it all :)

Reply:
Well, it's one line, but up to you. :)